### DISCUSSION TITLE
Display read-only value in the inspector

### DISCUSSION THREAD

**Discussion by reshin-pulapaddy** [clientIO/joint#2355] [Q&A]
### Introduction

Would it be possible to display key values inside the inspector element as read-only values?

example:

 model:         "model value'
session-id    "2345"
 
![presenation](https://github.com/clientIO/joint/assets/117301232/bd2241b8-7d08-4eaf-ad8d-e214ba13abdd)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
Yes. The inspector allows you to define a [custom field](https://resources.jointjs.com/docs/rappid/v3.7/ui.Inspector.html#ui.Inspector.custom-fields).
